packages/gamut-styles/src/utilities/boxShadow.ts

```diff
@@ -0,0 +1,37 @@
+import { css } from '@emotion/core';
+
+export type ShadowDepths = 1 | 2 | 3 | 4 | 5;
+
+const shadowsForDepth = {
+  1: {
+    offsets: [1, 3, 1, 3],
+    shades: [0.12, 0.24],
+  },
+  2: {
+    offsets: [3, 6, 3, 6],
+    shades: [0.19, 0.23],
+  },
+  3: {
+    offsets: [10, 20, 6, 6],
+    shades: [0.25, 0.22],
+  },
+  4: {
+    offsets: [14, 28, 10, 10],
+    shades: [0.25, 0.22],
+  },
+  5: {
+    offsets: [19, 38, 15, 12],
+    shades: [0.3, 0.22],
+  },
+};
+
+export const boxShadow = (depth = 1 as ShadowDepths) => {
```

Comment:
😉 

```suggestion
export const boxShadow = (depth: ShadowDepth = 1) => {
```

Explanation: TypeScript will infer a _primitive_ (`boolean`, `number`, `string`, ...) type when it sees a default value like that. So it was inferring `depth` to be of type `number`, even though you definitely wanted it to be the more specific `ShadowDepths` union of _literals_ (`false`, `1`, `"hi"`, ...).